runtime/browser/xwalk_runner.cc

```diff
@@ -176,6 +176,9 @@ void XWalkRunner::OnRenderProcessHostGone(content::RenderProcessHost* host) {
 void XWalkRunner::EnableRemoteDebugging(int port) {
   const char* local_ip = "0.0.0.0";
   if (port > 0 && port < 65535) {
+    // Ensure RemoteDebuggingServer release TCPServerSocket before applying for
+    // a new one even with same port number.
+    remote_debugging_server_.reset();
```

Comment:
would it be better to re-use the existing 'remote_debugging_server' rather than create new instance each time?